src/rules/requireJsdoc.js

```diff
@@ -140,11 +139,15 @@ const OPTIONS_SCHEMA = {
 };
 
 const getOption = (context, baseObject, option, key) => {
-  if (!_.has(context, `options[0][${option}][${key}]`)) {
-    return baseObject.properties[key].default;
+  if (context.options[0] && option in context.options[0] &&
+    // Todo: boolean shouldn't be returning property, but tests currently require
+    (typeof context.options[0][option] === 'boolean' ||
+    key in context.options[0][option])
+  ) {
+    return context.options[0][option][key];
   }
 
-  return context.options[0][option][key];
+  return baseObject.properties[key].default;
```

Comment:
I'm not aware of the context for these changes, but trust it makes sense.

Reply:
Yeah, although it is likely a bug (some issues reported which may suggest it), just using the boolean value of `context.options[0][option]` when found to be a boolean didn't work either, so would need investigation and a separate PR.

What this fix does is just restore the lodash behavior of avoiding erring when trying `in` checking on `context.options[0][option]` (as it will do when it is a boolean).